src/app/auth/login/page.tsx

```diff
@@ -29,10 +29,22 @@ const Login = () => {
     }
   };
 
+  const signInGoogle = async () => {
+    try {
+      await signIn('google');
+      toast('You have logged in successfully.', 'success');
+    } catch (err: any) {
+      toast(err.message, 'error');
+    }
+  };
+
   return (
     <main className="layout-auth">
       <h2 className="layout-auth__heading">Log In Form</h2>
       <LogInForm handleFormSubmit={handleFormSubmit} />
+      <button onClick={signInGoogle} className="form__button--submit">
+        Sign In with Google
```

Comment:
As you are using "Login", maybe we can make it 

```suggestion
        Log In with Google
```

![CleanShot 2023-08-30 at 17 32 59@2x](https://github.com/nimblehq/ic-nextjs-prabesh-dahal/assets/76901736/ba8dc63b-07ce-4459-a2aa-427bc7ec4994)